playwright.config.ts

```diff
@@ -0,0 +1,89 @@
+import { defineConfig, devices } from '@playwright/test';
+
+/**
+ * Read environment variables from file.
+ * https://github.com/motdotla/dotenv
+ */
+// require('dotenv').config();
```

Comment:
This comment-out intended? `.env` file can be useful.

Reply:
Not intentional, just not something I needed.  Will uncomment.